elements/pfe-card/src/pfe-card.scss

```diff
@@ -12,9 +12,13 @@ $LOCAL: card;
   // Combine above variables or allow single override point via variable
   --pfe-card--Padding: #{pfe-local(PaddingTop)} #{pfe-local(PaddingRight)} #{pfe-local(PaddingBottom)} #{pfe-local(PaddingLeft)};
 
-  // Internal spacing; elements inside the card (header, body, footer regions)
+
   --pfe-card--spacing: #{pfe-var(container-spacer)};
 
+  // Internal spacing; elements inside the card (header, body, footer regions)
+  --pfe-card--spacing--vertical: #{pfe-local(spacing)};
```

Comment:
Added a spacing variable specifically for the footer region so that it can be customized by designers.

Reply:
Good idea.